### ISSUE TITLE
Add Accessibility

### ISSUE THREAD

**Issue by jkeohan** [jkeohan/fe-mentor-job-listings#11]
Header
- [x] both desktop/mobile header images should be hidden (aria-hidden="true")  as they don't represent any relevant information

Search
- [x] Hide the clear button when search isn't active
- [ ] Reset focus to search box when a tag has been clicked

Job Listing 
- [x] Article should contain a hidden h1 with text (perhaps: frontend developer for shopify)
- [x] Hide dots using aria-hidden="true"
- [x] Add aria-labels to all postedAt, contract, location
- [x] Aria-label text for grouping of New/Featured

**Comment by jkeohan** [comment]
.
